packages/ui/src/proposals/components/ProposalList/ProposalList.tsx

```diff
@@ -22,7 +22,7 @@ export const ProposalList = ({ proposals, isPast }: ProposalListProps) => (
     </ListHeaders>
     <List>
       {proposals.map((proposal) => (
-        <ProposalListItem key={proposal.id} {...proposal} isPast={isPast} />
+        <ProposalListItem key={proposal.id} proposal={proposal} />
```

Comment:
The `isPast` should stay AFAICS

Reply:
OK I see that you've changed the ListItem logic there. ATM I'm torn between:

1.  Having it consistent with other lists (the prop is set from the outside)
2.  Having it automatically check

WDYT?

Reply:
From my point of view and as I would prefer to do: just check if it's past or not inside component. Because we can invoke isProposalActive in any place, so I would prefer to leave it as it is now.